generated/nidcpower/nidcpower.proto

```diff
@@ -2013,7 +1987,7 @@ message SetAttributeViInt64Request {
   nidevice_grpc.Session vi = 1;
   string channel_name = 2;
   NiDCPowerAttributes attribute_id = 3;
-  int64 attribute_value = 4;
+  int64 attribute_value_raw = 4;
```

Comment:
Note: This rename is to keep all SetAttribute APIs consistent. Even though this particular instance of SetAttribute doesn't have enum yet, adding `_raw` in the name makes it clear that users are supposed to pass in raw value.